### DISCUSSION TITLE
Mysql2::Error: Incorrect string value

### DISCUSSION THREAD

**Discussion by mattbaylor** [postalserver/postal#2837] [Help with using Postal]
We had a user send messages in Portuguese through the server and it seems to have choked on the encoding. Any ideas?

Mysql2::Error: Incorrect string value: '\xD3\xCA\xCF\xE4\xB2\xBB...' for column `e2postal-server-2`.`deliveries`.`output` at row 1
Support Ref: J-MULZLLPWNN

**Comment by willpower232** [comment]
Can you confirm what encoding the table has in your database?

**Comment by mattbaylor** [comment]
I just changed it to utf8mb4 with a collation of utf8mb4_unicode_ci. I have not restarted the db since that change however.

Matt Baylor

**Reply by willpower232** [reply]
You shouldn't need to restart the database or anything like that, are you able to send the messages now? What was it before?

**Comment by mattbaylor** [comment]
I unfortunately didn’t document it before I attempted the change, however all of the other server databases are already UTF8MB4, so I probably didn’t change anything...

Matt Baylor

**Comment by mattbaylor** [comment]
Messages are sending as normal, but there were about 5K messages that were showing they hadn't processed, however they had been sent as links were being clicked inside the message... very odd situation, but appears to be resolved.
